src/current/cockroachcloud/connect-to-your-cluster.md

```diff
@@ -42,16 +42,15 @@ Removing or adding an authorized network on your CockroachDB {{ site.data.produc
 Private connectivity allows you to establish SQL access to a CockroachDB {{ site.data.products.dedicated }} cluster entirely through cloud provider private infrastructure, without exposing the cluster to the public internet, affording enhanced security and performance.
 
 - Clusters deployed on GCP can connect privately using [GCP Private Service Connect (PSC)](#gcp-private-service-connect) or [GCP VPC peering](#gcp-vpc-peering). PSC allows you to selectively connect your cluster to a VPC within your Google Cloud project, while VPC Peering allows you to connect the Cockroach Cloud's VPC for your cluster to a VPC within your Google Cloud project.
-- Clusters deployed on AWS can connect privately using AWS PrivateLink, which allows you to connect Cockroach Cloud's VPC to a VPC within your AWS account.
+- Clusters deployed on AWS can connect privately using [AWS PrivateLink](#aws-privatelink), which allows you to connect Cockroach Cloud's VPC to a VPC within your AWS account.
```

Comment:
> which allows you to connect Cockroach Cloud's VPC to a VPC within your AWS account.

I have a slight issue with the wording here. At first blush, this looks like its describing VPC peering. Can we say "allows you to connect your cluster in Cockroach Cloud to a VPC within your AWS account"? Also, I think it should be "Cockroach**DB** Cloud"

Reply:
@adwittumuluri - Yep. we are considering the feature in (public) preview after the UI is available